workers/unity/ProjectSettings/ProjectVersionHash.txt

```diff
@@ -0,0 +1 @@
+4bc57476174c
```

Comment:
This is a **little** odd, though I'm not sure I have a better suggestion.

Reply:
I don't have another way to reconstruct the download url sadly

Reply:
What I meant was the location of this is odd - we've invented a new file that looks like Unity's, but it's not.

Reply:
I can change the location, but I'm not sure what would be a better place for it.
I can put in in `ci` since it is only used by the ci? And rename it to something like `UnityEditorHash.txt`
I'd rather not hardcode it into the script itself, as this makes updating it later more of a pain

Reply:
I would be fine with moving it to `ci` (or maybe `ci/includes`) and naming it `UnityEditorHash.txt`